cli/install/template.go

```diff
@@ -262,8 +252,6 @@ spec:
         image: {{.WebImage}}
         imagePullPolicy: {{.ImagePullPolicy}}
         args:
-        - "-addr=:8084"
-        - "-metrics-addr=:9994"
         - "-api-addr=api:8085"
```

Comment:
This isn't a result of your change, but I realized that this destination name isn't fully qualified, but we're using fully qualified names elsewhere. Can you change this to:

```
        - "-api-addr=api.{{.Namespace}}.svc.cluster.local:8085"
```